core/storage/storage_models_test.py

```diff
@@ -143,3 +152,57 @@ def test_model_classes_have_migrate_model(self):
             if (clazz.get_user_id_migration_policy() ==
                     base_models.USER_ID_MIGRATION_POLICY.CUSTOM):
                 self.assertTrue(hasattr(clazz, 'migrate_model'))
+
+    def test_get_models_which_should_be_exported(self):
+        """Ensure that the set of models to export is the set of models with
+        export policy CONTAINS_USER_DATA, and that all other models have
+        export policy NOT_APPLICABLE.
+        """
+        all_models = models.Registry.get_all_storage_model_classes()
+        models_with_export = (takeout_service
+                              .get_models_which_should_be_exported())
+        unimplemented_models = set()
+        for model in all_models:
+            export_policy = model.get_export_policy()
+            if model in models_with_export:
+                self.assertEqual(
+                    base_models.EXPORT_POLICY.CONTAINS_USER_DATA,
+                    export_policy
+                )
+            elif export_policy == base_models.EXPORT_POLICY.TO_BE_IMPLEMENTED:
+                unimplemented_models.add(model)
+            else:
+                self.assertEqual(
+                    base_models.EXPORT_POLICY.NOT_APPLICABLE,
+                    export_policy
+                )
+        # TODO(#8523): This number should be reduced to zero. This number
```

Comment:
This comment needs to be clarified; what "number" is it referring to?

Reply:
Done.+